worldengine/generation.py

```diff
@@ -188,25 +182,25 @@ def generate_world(w, step):
         return w
 
     # Precipitation with thresholds
-    PrecipitationSimulation().execute(w, seed)
+    PrecipitationSimulation().execute(w, seed * 13)
```

Comment:
What do you think about handing in a random value instead?

Maybe something like this:
```python
sub_seeds = numpy.random.randint(0, MAX_INT, size=100)  # create 100 random numbers; keep things deterministic in case of future expansions

PrecipitationSimulation().execute(w, sub_seeds[0])
ErosionSimulation().execute(w, sub_seeds[1])
...
```